scripts/ci/ci-run-sqllogic-tests.sh

```diff
@@ -7,23 +7,21 @@ set -e
 echo "Starting standalone DatabendQuery and DatabendMeta"
 ./scripts/ci/deploy/databend-query-standalone.sh
 
-SCRIPT_PATH="$(cd "$(dirname "$0")" >/dev/null 2>&1 && pwd)"
-cd "$SCRIPT_PATH/../../tests/logictest" || exit
-
-TEST_HANDLERS=${TEST_HANDLERS:-"mysql,http,clickhouse"}
+echo "Install rust"
+./scripts/setup/dev_setup.sh
```

Comment:
I need to use `Cargo` in the file, is it a good place to install `./scripts/setup/dev_setup.sh`? cc @everpcpc

Reply:
I think we can assume that the environment is already set up

Reply:
It's better to call `carge` directly and run `sqllogic-tests` with build-tools like unit test:

https://github.com/datafuselabs/databend/blob/main/.github/actions/test_unit/action.yml#L8-L27

Reply:
Since we don't need python anymore, we can remove the usage of `datafuselabs/build-tool:sqllogic`.

Reply:
Let me have a try. Thanks

Reply:
I'm willing to provide any help you need. I can update the workflow files if that blocks you. Just leave me a call~ :love_letter: